query-service/src/integrationTest/resources/configs/query-service/postgres/application.conf

```diff
@@ -15,8 +15,12 @@ service.config = {
   clients = [
     {
       type = postgres
-      connectionString = "jdbc:postgresql://localhost:5432/traceable?user=postgres&password=postgres"
-      connectionString = ${?POSTGRES_CONNECT_STR}
+      host = localhost
```

Comment:
For less envs + already prepared connection string, let's go with:
- connection string
- user
- password

What do you say?
```
connectionString = "jdbc:postgresql://localhost:5432/traceable
connectionString = ${?POSTGRES_CONNECT_STR}
user = postgres
user = ${?POSTGRES_USER}
password = postgres
password = ${?POSTGRES_PASSWROD}
```

Reply:
Sure. However user and password are optional for pinot. Correct?

Reply:
> Sure. However user and password are optional for pinot. Correct?

Yes, we only need this in Postgres for now.

Reply:
Done